src/views/product.js

```diff
@@ -16,24 +14,13 @@ export default class ProductView extends View {
   }
 
   /**
-   * check size of image until it is resolved, then set height of iframe.
+   * add event listener which triggers resize when the product image is loaded.
    */
-  resizeUntilLoaded() {
-    if (!this.iframe || !this.component.model.selectedVariantImage) {
-      return;
-    }
-    const img = this.wrapper.getElementsByClassName(this.component.classes.product.img)[0];
-    let intervals = 0;
-    if (img) {
-      const productResize = setInterval(() => {
```

Comment:
hi ok so.
I hate this. But it's the only way to get it to work consistently across all browsers. Might be worth re-testing (make sure you test FF, chrome on android, safari on iOS, with both portrait and landscape orientation images, with throttled network) to see if anything's changed, but I tried _so_ hard to avoid this. @richgilbank went through the same thing.